.github/workflows/slack-pr-notifications.yaml

```
@@ -0,0 +1,754 @@
+            echo "⚠️ Unknown event type: $EVENT_TYPE. Skipping notification."
+            exit 0
+          fi
+
+          echo "SKIP_NOTIFICATION=false" >> $GITHUB_ENV
+          echo "SLACK_COLOR=$SLACK_COLOR" >> $GITHUB_ENV
+          echo "TITLE=$TITLE" >> $GITHUB_ENV
+          echo "DESCRIPTION=$DESCRIPTION" >> $GITHUB_ENV
+
+      - name: Send Slack notification
+        if: env.SKIP_NOTIFICATION != 'true' && github.event.pull_request.number != '' && github.event.pull_request.title != ''
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)